src/components/SigningModals/TransactionStepper/useStepperLayout.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useRef, useState } from 'react'
+import useMeasure from 'react-use-measure'
+
+function useStepperLayout() {
+  const [outerBoundsRef, outerBounds] = useMeasure()
+  const innerBoundsRef = useRef()
+  const [innerBounds, setInnerBounds] = useState(null)
+  const [measuring, setMeasuring] = useState(true)
+
+  // First render must always be in "multiple" mode so that our measurement reference point is accurate
+  const [layout, setLayout] = useState('multiple')
+
+  useEffect(() => {
+    const outerMeasured = outerBounds.width && outerBounds.width > 0
```

Comment:
On initial render the width has yet to be determined by resize obs, this is expected behaviour.